week_03/codewars/dglee/Shortest_Word.java

```diff
@@ -0,0 +1,15 @@
+package Cote_Package;
+
+public class Shortest_Word {
+    public static int findShort(String s) {
+        String[] array = s.split(" ");
+        int short_length = array[0].length();
+        
+        for (int i = 1; i < array.length; i++)
+            short_length = (array[i].length() > short_length) ? short_length : array[i].length();
```

Comment:
```suggestion
            int length_of_word = array[i].length()
            short_length = (length_of_word > short_length) ? short_length : length_of_word;
```
요렇게 중복되는 값을 변수로 extract 해서 쓸 수 있을 것 같애(내가 자바를 잘 몰라서 얼추 맞춰서 썼느데 틀릴 수도 있으니.. 감안해서 보시길)